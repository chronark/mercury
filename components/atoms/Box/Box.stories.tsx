/* eslint-disable */
import React from "react"
import { storiesOf } from "@storybook/react"
import { Box } from "./Box"

storiesOf("atoms/Box", module).add("default", () => (
  <Box>
    <div className="p-4">Something inside</div>
  </Box>
))
