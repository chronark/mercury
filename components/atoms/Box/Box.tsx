import React from "react"

export interface BoxProps {
  className?: string
}

export const Box: React.FC<BoxProps> = ({ children, className }) => (
  <div className="p-8 bg-white rounded shadow-2xl">
    <div className={className}>{children}</div>
  </div>
)
