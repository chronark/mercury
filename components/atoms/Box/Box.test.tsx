import React from "react"
import renderer from "react-test-renderer"
import { Box } from "./Box"

it("renders correctly", () => {
  const tree = renderer
    .create(
      <Box>
        <span>Hello World</span>
      </Box>,
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
