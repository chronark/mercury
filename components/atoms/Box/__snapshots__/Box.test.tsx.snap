// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="p-8 bg-white rounded shadow-2xl"
>
  <div>
    <span>
      Hello World
    </span>
  </div>
</div>
`;
